flex/exceptions.py

```diff
@@ -103,3 +103,6 @@ class NoParameterFound(ValueError):
 
 class MultipleParametersFound(ValueError):
     pass
+
+class DataCouldNotBeParsed(ValueError):
+    pass
```

Comment:
I'm not sure this is the best approach, it means that either `json.JSONDecodeError` or `flex.http.JSONDecodeError` could be raised, I think it would be better if it was a single exception.

One thing we could do is depend on `simplejson` which has a `JSONDecodeError` itself?